src/config/dynamic-theme-fixes.config

```diff
@@ -4186,6 +4186,21 @@ INVERT
 
 ================================
 
+campus.datacamp.com
+
+CSS
+.css-hu6jey code {
+     mix-blend-mode: normal !important;
+     color: var(--darkreader-neutral-text) !important;
+     
+}
+.css-hu6jey pre {
```

Comment:
Can you share specyfic URL to the course where pre is needed to be fixed too?
I used https://campus.datacamp.com/courses/intro-to-python-for-data-science/chapter-1-python-basics?ex=3 and without this class it worked fine with only .css-hu6jey code fix.

Reply:
you can check the difference here :

https://campus.datacamp.com/courses/text-mining-with-bag-of-words-in-r/jumping-into-text-mining-with-bag-of-words?ex=3